src/selmer/filters.clj

```diff
@@ -77,6 +77,16 @@ map. The rest of the arguments are optional and are always strings."
             :str
             str
 
+            ;;; Like the subs, only adds rest if s is modified
```

Comment:
The functionality for `rest` is not in `clojure.core/subs`. Shouldn't that be documented?